src/libs/portfolio/interfaces.ts

```diff
@@ -30,6 +31,7 @@ export type TokenResult = {
     isFeeToken: boolean
     isHidden?: boolean
     isCustom?: boolean
+    isSuspected?: IsSuspectedType
```

Comment:
this one is a bit bh, but variables starting with `is` are expected to be of boolean type. rename to suspectedType (like rewardsType)